plugin/completion/bin_complete.py

```diff
@@ -116,7 +116,7 @@ def exists_for_view(self, view_id):
         Returns:
             bool: compile flags exist for this view
         """
-        if view_id not in self.flags_dict:
+        if self.flags_dict.get(view_id) is None:
```

Comment:
View ids are removed from flags_dict this way: flags_dict[view_id] = []`
so checking 'view_id' not in flags_dict` will not return None after view id was removed.
So that change makes sense to me.